pkg/registry/registry.go

```diff
@@ -113,16 +123,41 @@ func (r *Registry) Push(ctx context.Context, repo, tag string) error {
 func (r *Registry) RemoveImage(ctx context.Context, repo, tag string) error {
 	image := fmt.Sprintf("%s/%s:%s", r.name, repo, tag)
 
+	runningImages, err := r.listRunningImages()
+	if err != nil {
+		return microerror.Mask(err)
+	}
+	if _, ok := runningImages[image]; ok {
+		// Host is currently running this particular image; Trying to remove it
+		// would cause errors.
+		return nil
+	}
+
 	args := []string{"rmi", image}
 
-	err := executeCmd(dockerBinaryName, args)
+	err = executeCmd(dockerBinaryName, args)
```

Comment:
Do you think I should change existing `executeCmd` calls, since we have `r.dockerClient` now?

Reply:
I tried that as well but `github.com/docker/docker v1.13.1` - looks like the last commit is from Feb 2017 https://github.com/moby/moby/tree/v1.13.1

There were many bugs fixed since then. I understand that listing/pushing/pulling images is pretty solid but I'm still not sure if this is good idea to use that dependency at all.

Reply:
Why did you decide against binary output?

Reply:
Oh, right. It's because they moved to `v19.X.Y` versioning. I can switch to a more recent client, no problem.

I didn't want to parse this kind of output: 
```
"CONTAINER ID        IMAGE               COMMAND                  CREATED             STATUS              PORTS               NAMES\n25c103446779        alpine:3            \"/bin/sh -c 'sleep 3…\"   40 minutes ago      Up 40 minutes                           serene_wiles\n"
```

If you feel like using the client is overkill or breaks an assumption you guys made earlier, I can totally parse `docker ps` outpuy.

Reply:
There is `--format string   Pretty-print containers using a Go template` option.

Reply:
```
$ docker ps -a --format '{{ .Image }}'
quay.io/giantswarm/hugo:v0.70.0
```

Reply:
Discussed in Meet, we don't trust docker client enough. I'll go back to parsing `docker ps` output.